### ISSUE TITLE
Spack Package for MET

### ISSUE THREAD

**Issue by rbeucher** [ACCESS-NRI/spack_packages#29]
I need to install METPlus which is a wrapper on MET. MET needs to be installed on NCI Gadi and I was wondering if someone could have a look at that.

See https://github.com/ACCESS-NRI/METPlus-workflow/issues/1

Thanks!

Romain

**Comment by rbeucher** [comment]
@harshula would you have time to get me through a spark crash course so I can see how much work is needed there?

**Comment by rbeucher** [comment]
After a discussion with @aidanheerdegen, I think having someone from MED pair up with someone from the release team on this would be good. @max-anu ?

Here is the link to the MET documentation. METPlus is the python wrapper that we will use to call MET.

**Comment by harshula** [comment]
Had a chat about this last week. Looks like the dependencies are already in Spack. If the METPlus build system is good, a simple SPD should be sufficient.

**Comment by rbeucher** [comment]
Thanks @harshula, Would be great if you could show @max-anu how it is done.

**Comment by harshula** [comment]
Hi @max-anu ,
* The quickest way to get a feel for Spack is to build access-om2. The instructions are: https://github.com/ACCESS-NRI/ACCESS-OM/issues/6#issuecomment-1637387506
* Then, have a look at the spack_packages git repository that was cloned in the aforementioned instructions. In particular have a look at libaccessom2 and oasis3-mct. They represent two extremes of what needs to be done if the source tarball has a good vs bad build system.
* Make sure you understand the source tarball built system well before starting to work on the Spack Package Definition (SPD). AKA package.py.

**Comment by aidanheerdegen** [comment]
We should also check this hasn't already been done by the Bureau.  @scottwales have you built METplus with spack already?

**Comment by ScottWales** [comment]
Yes, there's a recipe in the spack-environments repository, and a container available on Gadi - https://forum.access-hive.org.au/t/metplus-5-0-available-at-nci/810

**Comment by aidanheerdegen** [comment]
Thanks @ScottWales

**Comment by rbeucher** [comment]
Thanks @ScottWales.

**Comment by harshula** [comment]
Hi @rbeucher , Can this issue be closed as completed?
